# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/vm/TypedArrayObject.cpp
# Commit: 16695064a637
# Full Hash: 16695064a637776379f4cb777771087cf12c0760
# Author: Steve Fink <sfink@mozilla.com>
# Date: 2021-08-13 09:27:46
# Regressor Bug: 1720422
# File Overlap Count: 1
# Description:
#   Bug 1720422 - Convert the JS_Is(type)Array APIs r=jonco
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D119860
# ==============================================================================

diff -r b5bafd6f08d3 -r 16695064a637 js/src/vm/TypedArrayObject.cpp
--- a/js/src/vm/TypedArrayObject.cpp	Thu Aug 12 20:44:17 2021 +0000
+++ b/js/src/vm/TypedArrayObject.cpp	Thu Aug 12 20:44:17 2021 +0000
@@ -2643,10 +2643,6 @@
     return obj;                                                               \
   }                                                                           \
                                                                               \
-  JS_PUBLIC_API bool JS_Is##Name##Array(JSObject* obj) {                      \
-    return js::Unwrap##Name##Array(obj) != nullptr;                           \
-  }                                                                           \
-                                                                              \
   JS_PUBLIC_API ExternalType* JS_Get##Name##ArrayLengthAndData(               \
       JSObject* obj, size_t* length, bool* isSharedMemory,                    \
       const JS::AutoRequireNoGC& nogc) {                                      \
